diff -ur "eccodes-2.23.0-Source (copy 1)/src/grib_optimize_decimal_factor.c" eccodes-2.23.0-Source/src/grib_optimize_decimal_factor.c
--- "eccodes-2.23.0-Source (copy 1)/src/grib_optimize_decimal_factor.c"	2021-08-25 11:35:41.000000000 +0200
+++ eccodes-2.23.0-Source/src/grib_optimize_decimal_factor.c	2021-11-12 11:20:43.331305877 +0100
@@ -28,9 +28,7 @@
     return (int)(log(DBL_MAX) / log(10)) - 1;
 }
 
-#ifdef ECCODES_ON_WINDOWS
 #define log2(a) (log(a) / 1.44269504088896340736)
-#endif
 
 static void factec(int* krep, const double pa, const int knbit, const long kdec, const int range, long* ke, int* knutil)
 {
